awscli/customizations/s3/s3handler.py

```diff
@@ -186,7 +187,8 @@ def _enqueue_tasks(self, files):
                                          message=warning_message)
                 self.result_queue.put(warning)
             # Warn and skip over glacier incompatible tasks.
-            elif not filename.is_glacier_compatible():
+            elif not filename.is_glacier_compatible(
```

Comment:
Good point. Passing it through made sense when I was forcing a HEAD request, but condensing it makes more sense now.